code/modules/chemistry/tools/dispensers.dm

```diff
@@ -730,55 +742,72 @@ TYPEINFO(/obj/reagent_dispensers/watertank/fountain)
 				user.u_equip(W)
 				W.dropped(user)
 				qdel(W)
+				playsound(src.loc, 'sound/effects/bubbles_short.ogg', 30, 1)
 				return
 			else  ..()
-		else ..()
+		// create feedback for items which don't produce attack messages
+		if (W && (W.flags & SUPPRESSATTACK))
+			if (isfull)
+				boutput(user, SPAN_ALERT("[src] is already full."))
+			else
+				boutput(user, SPAN_ALERT("Can't brew anything from [W]."))
+		..()
 
 	MouseDrop_T(atom/movable/O as mob|obj, mob/user as mob)
 		if (!isliving(user))
-			user.show_text("It's probably a bit too late for you to drink your problems away.", "red")
+			boutput(user, SPAN_ALERT("It's probably a bit too late for you to drink your problems away!"))
 			return
 		if (BOUNDS_DIST(user, src) > 0)
-			user.show_text("You need to move closer to [src] to do that.", "red")
+			// You have to be adjacent to the still
+			boutput(user, SPAN_ALERT("You need to move closer to [src] to do that."))
 			return
-		if (BOUNDS_DIST(O, src) > 0 || BOUNDS_DIST(O, user) > 0)
-			user.show_text("[O] is too far away to load into [src]!", "red")
+		if (BOUNDS_DIST(O, user) > 0)
+			// You have to be adjacent to the brewables also
+			boutput(user, SPAN_ALERT("[O] is too far away to load into [src]!"))
 			return
-
+		// loading from crate
 		if (istype(O, /obj/storage/crate/))
-			user.visible_message(SPAN_NOTICE("[user] loads [O]'s contents into [src]!"),\
-			SPAN_NOTICE("You load [O]'s contents into [src]!"))
+			user.visible_message(SPAN_NOTICE("[user] begins to charge [src] with [O]'s contents!"))
 			var/amtload = 0
+			var/staystill = user.loc
 			for (var/obj/item/P in O.contents)
+				if (user.loc != staystill) break
 				if (src.reagents.is_full())
-					user.show_text("[src] is full!", "red")
+					boutput(user, SPAN_ALERT("[src] is full!"))
 					break
 				if (src.brew(P))
 					amtload++
 					qdel(P)
+					playsound(src.loc, 'sound/effects/bubbles_short.ogg', 30, 1)
+					sleep(0.2 SECONDS) // faster than from the floor because you organised your crate well
```

Comment:
If you want the action to have a specific cooldown, then I think making an actionbar is better than having arbitrary sleep times, especially if you vary them depending on if it's a crate or not. Also nothing in the game behaves this way when being loaded from a crate, might warrant a message to the player.

Reply:
That's fair, I just copied the functionality from the way the compost bin is coded and changed the crate time because I thought it was cute. I'll probably just try to find how other crates work rather than add an action bar since I feel like an action bar is a bigger departure from expectation.